docker-compose.yml

```diff
@@ -1,32 +1,61 @@
-version: '2'
+version: '3.8'
+
 services:
-  web:
-    build:
-      dockerfile: Dockerfile.backend
-      context: .
-    command: yarn serve
+  backend:
+    restart: always 
+    build: 
+      context: ./packages/backendv2
+      dockerfile: development.Dockerfile
     ports:
-      - "3000:3000"
+      - 3003:3003
     depends_on:
       - redis
       - postgres
     environment:
-      REDIS_HOST: redis
-      REDIS_PORT: 6379
-      DB_HOST: postgres
-      DB_USER: postgres
-      DB_PASSWORD: postgres
-  dashboard:
+      - REDIS_HOST=redis
+      - REDIS_PORT=6379
+      - DB_HOST=postgres
+      - DB_USER=postgres
+      - DB_PASSWORD=postgres
+  
+  example:
+    restart: always
     build:
-      dockerfile: Dockerfile.dashboard
-      context: .
+      context: ./packages/moocfi-quizzes
+      dockerfile: development.Dockerfile
     ports:
-      - "5001:80"
-  redis:
-    image: redis:5-alpine
-    container_name: redis
+      - 1234:1234
+    depends_on:
+      - backend
+
+  dashboard:
+    restart: always
+    build: 
+      context: ./packages/quizzes-dashboard
+      dockerfile: development.Dockerfile
+    volumes:
+      - ./packages/quizzes-dashboard:/quizzes/dashboard
+    depends_on:
+      - backend
+    ports:
+      - 5000:5000
+
   postgres:
     image: postgres:11-alpine
+    restart: always
+    ports:
+      - 5432:5432
     environment:
-      POSTGRES_USER: postgres
-      POSTGRES_PASSWORD: postgres
+      - POSTGRES_DB=postgres
+      - POSTGRES_USER=postgres
+      - POSTGRES_PASSWORD=postgres
+    volumes:
+      - postgres:/var/lib/postgresql/data
+
+  redis:
+    image: redis:5-alpine
+    container_name: redis
+    restart: always 
+
+volumes:
+  postgres: ~
```

Comment:
I think this key should not have a value. Upon seeing this, I initially mistakenly assumed that this would cause the volume to be persisted to the home directory, but after looking at the docs, I think that the tilde is just ignored.

Anyway, I think this would be less confusing without the tilde.